### DISCUSSION TITLE
Step by Step Documentation + Look & Feel + User Experience

### DISCUSSION THREAD

**Discussion by aqoamann** [mkaring/ConfuserEx#462] [Q&A]
Hi! Thank you all for your efforts 👍
While iterating through the [list of obfuscators for .NET](https://github.com/NotPrab/.NET-Obfuscator) and trying them, I came to ConfuserEx2 which actually was the only one that worked 🥇 

1. I could not find any step by step documentation (I am using ver. 1.7.0-alpha). If you like, I could write one (MD file with screenshots). Do you?

2. IMHO the design is a little too dark, far from "standard" and a little bit "nerdy"... the default look & feel would make it easier for new users (like me) to get familiar with it (since there is no docu yet).

3. UX: since there is no documentation, some ToolTips would help to 1) navigate and 2) find out what info put in which TextBox.

4. ?BUG? In the "Settings" TabItem, the ListBox "Rules:" does show only "true" for everything. Multiple "true" items in the ListBox are  confusing. Would be better if one could rename them.

Kindly regards

**Comment by mkaring** [comment]
Hello,

1. Sure, go ahead.
2. The UI is already pretty old. There will be a new one some time in the future. But no date is set yet. We'll need a new one anyway to correctly support cross platform usage.
3. Same as 2. :wink:
4. The `true` is actually the pattern expression of the rule. There shouldn't be multiple `true` lines in a correctly setup. It should look like this for example: ![image](https://user-images.githubusercontent.com/842566/155015128-90b8f2bb-9a26-43b0-aefa-a30b8e97a90d.png)

Best regards,
Martin
